diff --git a/ports/curl/CONTROL b/ports/curl/CONTROL
index 9ae7e7e5..e6fe237d 100644
--- a/ports/curl/CONTROL
+++ b/ports/curl/CONTROL
@@ -1,4 +1,4 @@
 Source: curl
 Version: 7.51.0-3
-Build-Depends: zlib, openssl, libssh2
+Build-Depends: openssl
 Description: A library for transferring data with URLs
diff --git a/ports/curl/portfile.cmake b/ports/curl/portfile.cmake
index 35bfbd59..0191abb2 100644
--- a/ports/curl/portfile.cmake
+++ b/ports/curl/portfile.cmake
@@ -43,6 +43,9 @@ else()
             -DBUILD_TESTING=OFF
             -DBUILD_CURL_EXE=OFF
             -DENABLE_MANUAL=OFF
+            -DCMAKE_USE_OPENSSL=ON
+            -DCURL_ZLIB=OFF
+            -DCMAKE_USE_LIBSSH2=OFF
             -DCURL_STATICLIB=${CURL_STATICLIB}
         OPTIONS_DEBUG
             -DENABLE_DEBUG=ON
			 